pkg/remote/client_test.go

```diff
@@ -3,15 +3,18 @@ package remote
 import (
 	"context"
 	"io"
+	"math"
 	"net/http"
 	"net/http/httptest"
 	"net/url"
 	"testing"
 	"time"
 
+	"github.com/golang/snappy"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	prompb "go.buf.build/grpc/go/prometheus/prometheus"
+	"google.golang.org/protobuf/proto"
 )
 
 func TestNewWrtiteClient(t *testing.T) {
```

Comment:
Not relevant to this PR, but:

```suggestion
func TestNewWriteClient(t *testing.T) {
```